.changeset/quick-buses-obey.md

```diff
@@ -0,0 +1,17 @@
+---
+'@graphql-tools/executor-http': major
+'@graphql-mesh/transport-http-callback': minor
+'@graphql-hive/gateway-runtime': minor
+'@graphql-tools/federation': patch
+---
+
+- **BREAKING**: HTTP Executor no longer takes `serviceName` as an option.
```

Comment:
Any reason for the breaking change? or is it just internal and GW is not affected?

Reply:
No breaking changes for the GW